specification/ai/Azure.AI.Unified/evaluations/models.tsp

```diff
@@ -0,0 +1,90 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "../common/models.tsp";
+import "../main.tsp";
+import "@typespec/openapi";
+
+using TypeSpec.OpenAPI;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Azure.AI.Unified;
+
+@doc("Evaluator Configuration")
+model EvaluatorConfiguration {
+  @doc("Identifier of the evaluator.")
+  id: string;
+
+  @doc("Initialization parameters of the evaluator.")
+  initParams?: Record<unknown>;
+
+  @doc("Data parameters of the evaluator.")
+  dataMapping?: Record<string>;
+}
+
+#suppress "@azure-tools/typespec-azure-core/no-string-discriminator"
+@doc("Abstract data class.")
+@discriminator("type")
+model InputData {
+  @doc("Type of the data")
+  type: string;
+}
+
+@doc("Dataset as source for evaluation.")
+model Dataset extends InputData {
+  type: "dataset";
+
+  @doc("Evaluation input data")
+  id: string;
+}
+
+@doc("Evaluation Definition")
+model Evaluation {
+  @doc("Identifier of the evaluation.")
+  id?: string;
+
+  @doc("Data for evaluation.")
+  data: InputData;
+
+  @doc("Update stage to 'Archive' to archive the asset. Default is Development, which means the asset is under development.")
+  displayName?: string;
+
+  @doc("Description of the evaluation. It can be used to store additional information about the evaluation and is mutable.")
+  description?: string;
+
+  @doc("Metadata containing createdBy and modifiedBy information.")
+  @visibility("read")
+  systemData?: SystemData;
```

Comment:
Is this taken directly from control plane? `systemData` is not a good name for the property.